package.json

```diff
@@ -25,7 +26,7 @@
     "relay": "node scripts/relayCompiler.js",
     "sentry": "scripts/sentry.sh",
     "start": "scripts/start.sh",
-    "start:prod": "NODE_ENV=production node -r dotenv/config --max_old_space_size=4096 ./src --verbose --debugProd",
+    "start:prod": "NODE_ENV=production yarn start",
```

Comment:
I wonder if it would be worth adding `yarn assets && yarn build:server && ...` to this

Reply:
It could lead to less confusion. I feel like once someone runs this once they'll know to run `NODE_ENV=production yarn start`.

Reply:
Updated 👍